streetteam/docker-images
# install heroku plugin on deploy

We need heroku plugins on deploy as well
to test:
- run `docker build circleci-deploy`
- run `docker run -it {IMAGENAME}`
- when you are in the container, run `heroku plugins`, you should see both `heroku-repo` and `heroku-releases-retry`
